test/test.js

```diff
@@ -37,4 +37,8 @@ describe('postcss-at2x', () => {
   it('should process image and add background size', (done) => {
     runTest('read-background-size.css', {detectImageSize: true}, done);
   });
+
+  it('should resolve image path with custom function', (done) => {
+    runTest('resolve-image-path.css', {resolveImagePath: (value) => {return path.resolve(process.cwd(), value);}}, done);
+  });
```

Comment:
Hmm, something is up with Jest on your system. I just pulled the branch and ran the tests and it wrote the snapshot.

I'm using Node 6 but the tests run on Node 4 on CI so it should be fine